.circleci/config.yml

```diff
@@ -26,11 +26,23 @@ jobs:
           name: run tests
           command: npm test
 
+
+  develop_deploy:
```

Comment:
separate deploys for develop, staging, and production.

the repeated steps can be DRY'd up later